### DISCUSSION TITLE
小火箭百度贴吧去广告无效，请教一下是否这样配置的？

### DISCUSSION THREAD

**Discussion by AkimotoTeng** [blackmatrix7/ios_rule_script#1043] [Q&A]
小火箭配置文件里新加了贴吧分流规则，但是打开贴吧还是有广告，而且加载很慢，请问一下是我哪里还没设置好吗？

[https://github.com/blackmatrix7/ios_rule_script/tree/master/rule/Shadowrocket/BaiDuTieBa](url)

![default](https://github.com/blackmatrix7/ios_rule_script/assets/148468651/f588e937-98e4-4f12-aec6-fcd7edf78d08)
![default1](https://github.com/blackmatrix7/ios_rule_script/assets/148468651/1f0613b5-901d-4fe7-96ec-1b7943c86f93)
![default3](https://github.com/blackmatrix7/ios_rule_script/assets/148468651/0fa9be78-5e36-4abb-ab75-df7c676381ab)

**Comment by gaozx5519** [comment]
我也想知道小火箭咋去广告  我我只会用qx
